[INFO   ] Logger: Record log in /storage/emulated/0/as-app/.kivy/logs/kivy_24-12-24_31.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 190 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2 (img_pil, img_ffpyplayer ignored)
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 build 1.13@5776728'>
[INFO   ] GL: OpenGL vendor <b'Imagination Technologies'>
[INFO   ] GL: OpenGL renderer <b'PowerVR Rogue GE8320'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <8192>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Text: Provider: sdl2
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 240, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:            ^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "<string>", line 18, in <module>
[WARNING] stderr: NameError: name 'cente' is not defined
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 694, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:                    ^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 243, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "<inline>", line 18:
[WARNING] stderr: ...
[WARNING] stderr:      16:    spacing: "15dp"
[WARNING] stderr:      17:    padding: "10dp"
[WARNING] stderr: >>   18:    pos_hint: cente
[WARNING] stderr:      19:
[WARNING] stderr:      20:    Label:
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'cente' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 240, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:            ^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "<string>", line 18, in <module>
[WARNING] stderr: 
[WARNING] stderr: 
[WARNING] stderr: During handling of the above exception, another exception occurred:
[WARNING] stderr: 
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/as-app/main.py", line 20, in <module>
[WARNING] stderr:     ASMobileApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 955, in run
[WARNING] stderr:     self._run_prepare()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 925, in _run_prepare
[WARNING] stderr:     root = self.build()
[WARNING] stderr:            ^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/as-app/main.py", line 15, in build
[WARNING] stderr:     self.main_screen_manager = MainScreenManager()
[WARNING] stderr:                                ^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/storage/emulated/0/as-app/views/mainsm.py", line 10, in __init__
[WARNING] stderr:     self.add_widget(LogInScreen(name="login-screen"))
[WARNING] stderr:                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/relativelayout.py", line 274, in __init__
[WARNING] stderr:     super(RelativeLayout, self).__init__(**kw)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/floatlayout.py", line 65, in __init__
[WARNING] stderr:     super(FloatLayout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/layout.py", line 76, in __init__
[WARNING] stderr:     super(Layout, self).__init__(**kwargs)
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/widget.py", line 366, in __init__
[WARNING] stderr:     self.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 540, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 662, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 662, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 662, in _apply_rule
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 660, in _apply_rule
[WARNING] stderr:     child.apply_class_lang_rules(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/widget.py", line 470, in apply_class_lang_rules
[WARNING] stderr:     Builder.apply(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 540, in apply
[WARNING] stderr:     self._apply_rule(
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 709, in _apply_rule
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: kivy.lang.builder.BuilderException: Parser: File "<inline>", line 18:
[WARNING] stderr: ...
[WARNING] stderr:      16:    spacing: "15dp"
[WARNING] stderr:      17:    padding: "10dp"
[WARNING] stderr: >>   18:    pos_hint: cente
[WARNING] stderr:      19:
[WARNING] stderr:      20:    Label:
[WARNING] stderr: ...
[WARNING] stderr: BuilderException: Parser: File "<inline>", line 18:
[WARNING] stderr: ...
[WARNING] stderr:      16:    spacing: "15dp"
[WARNING] stderr:      17:    padding: "10dp"
[WARNING] stderr: >>   18:    pos_hint: cente
[WARNING] stderr:      19:
[WARNING] stderr:      20:    Label:
[WARNING] stderr: ...
[WARNING] stderr: NameError: name 'cente' is not defined
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 240, in create_handler
[WARNING] stderr:     return eval(value, idmap), bound_list
[WARNING] stderr:            ^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "<string>", line 18, in <module>
[WARNING] stderr: 
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 694, in _apply_rule
[WARNING] stderr:     value, bound = create_handler(
[WARNING] stderr:                    ^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 243, in create_handler
[WARNING] stderr:     raise BuilderException(rule.ctx, rule.line,
[WARNING] stderr: 
